addons/crm/crm_tip_data.xml

```diff
@@ -14,38 +14,37 @@
         </record>
 
         <record model="web.tip" id="crm_tip_2">
-            <field name="title"></field>
-            <field name="description">By clicking here, you can keep a track of your sales potential concerning this customer. You be able to plan meeting and calls related to the opportunities.</field>
+            <field name="title">Nice</field>
+            <field name="description">Here is your partner ! Keep track of your opportunities concerning this customer here. Simple and easy.</field>
             <field name="action_id" ref="base.action_partner_form"/>
             <field name="model">res.partner</field>
             <field name="mode">form</field>
             <field name="trigger_selector">.oe_form_buttons_view:visible,div.oe_right.oe_button_box > button</field>
-            <field name="highlight_selector">div.oe_right.oe_button_box:visible > button:nth-child(1)</field>
+            <field name="highlight_selector">div.oe_right.oe_button_box:visible > #total_opportunity</field>
             <field name="placement">bottom</field>
         </record>
 
         <record model="web.tip" id="crm_tip_3">
-            <field name="title"></field>
-            <field name="description">This where you navigate through your process. Click here to go back to where you came from.</field>
+            <field name="title">Update your progress</field>
+            <field name="description">Here you can also change the status of the opportunity.</field>
             <field name="model">crm.lead</field>
             <field name="type">opportunity</field>
             <field name="mode">form</field>
             <field name="trigger_selector">.oe_form_buttons_view:visible,ol.breadcrumb a</field>
-            <field name="highlight_selector">ol.breadcrumb:visible a:last</field>
-            <field name="placement">bottom</field>
+            <field name="highlight_selector">.oe_form_field_status</field>
```

Comment:
Don't break the flux please. Here the intention to explain the breadcrumb and to help the user during the process. 

If the user changes the Opportunity's stage, after we will propose him to drag and drop the task to change the stage => Redundancy